i18n/zh-CN/docusaurus-plugin-content-docs/current/user-guide/3-development.md

```diff
@@ -4,93 +4,92 @@ title: '开发环境'
 sidebar_position: 3
 ---
 
-> [StreamPark](https://github.com/apache/incubator-streampark) 遵循 Apache-2.0 开源协议，将会是个长期更新的活跃项目，欢迎大家提交 [PR](https://github.com/apache/incubator-streampark/pulls) 或 [ISSUE](https://github.com/apache/incubator-streampark/issues/new/choose) 喜欢请给个 [Star](https://github.com/apache/incubator-streampark/stargazers) 您的支持是我们最大的动力。 该项目自开源以来就受到不少朋友的关注和认可，表示感谢。已陆续有来自金融，数据分析，车联网，智能广告，地产等公司的朋友在使用，也不乏来自一线大厂的朋友在使用。
-同时 StreamPark 社区是一个非常开放,相互协助,尊重人才的社区。我们也非常欢迎更多的开发者加入一块贡献，不只是代码的贡献，还寻求使用文档，体验报告，问答等方面的贡献。
+> [StreamPark](https://github.com/apache/incubator-streampark) 遵循 Apache-2.0 开源协议，将会是个长期更新的活跃项目，欢迎大家提交 [PR](https://github.com/apache/incubator-streampark/pulls) 或 [ISSUE](https://github.com/apache/incubator-streampark/issues/new/choose) 喜欢请给个 [Star](https://github.com/apache/incubator-streampark/stargazers) 您的支持是我们最大的动力。该项目自开源以来就受到不少朋友的关注和认可，表示感谢。已陆续有来自金融，数据分析，车联网，智能广告，地产等公司的朋友在使用，也不乏来自一线大厂的朋友在使用。
```

Comment:
thanks for your contribution, streampark is already an apache incubation project, this paragraph is no longer appropriate, I think about how to change it😭